containers/dotnetcore-mssql/.devcontainer/mssql/postCreateCommand.sh

```diff
@@ -0,0 +1,80 @@
+dacpac="false"
+sqlfiles="false"
+SApassword=$1
+dacpath=$2
+sqlpath=$3
+
+echo "SELECT * FROM SYS.DATABASES" | dd of=testsqlconnection.sql
+for i in {1..60};
+do
+    /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P $SApassword -d master -i testsqlconnection.sql
+    if [ $? -eq 0 ]
+    then
+        echo "sql server ready"
+        break
+    else
+        echo "not ready yet..."
+        sleep 1
+    fi
+done
+rm testsqlconnection.sql
+
+for f in $dacpath/*
+do
+    if [ $f == $dacpath/*".dacpac" ]
+    then
+        dacpac="true"
+        echo "found dacpac $f"
+    fi
+done
+
+for f in $sqlpath/*
+do
+    if [ $f == $sqlpath/*".sql" ]
+    then
+        sqlfiles="true"
+        echo "found sql file $f"
+    fi
+done
+
+if [ $sqlfiles == "true" ]
+then
+    echo "installing mssql-tools"
```

Comment:
This part could be in the Dockerfile for the main container rather than the SQL one.

Reply:
Splitting this between installation and execution makes sense.  Ideally we'd have a sqltools container image for simplicity, but we're not there yet.